### PR TITLE
feat: Add Embed command response for the Role group as well as help s…

### PR DESCRIPTION
**Pull request by krustykarbs**
…ummary for role create arguments

### CODE REVIEW

**Comment by sabihoshi** [comment]
I prefer to use `Format.Bold` for formatting instead of `**`

**Comment by krustykarbs** [comment]
> I prefer to use `Format.Bold` for formatting instead of `**`

50acb30 should satisfy this

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 1]
```diff
@@ -110,19 +138,29 @@ public async Task DeleteRolesAsync(params IRole[] roles)
                 await role.DeleteAsync();
             }
 
-            await Context.Message.AddReactionAsync(new Emoji("✅"));
+            var embed = new EmbedBuilder()
+                .WithDescription($"Deleted the following role(s): {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} from the guild.")
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:142]: `roles.Humanize()`

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 2]
```diff
@@ -110,19 +138,29 @@ public async Task DeleteRolesAsync(params IRole[] roles)
...
 
         [Command("remove")]
         [Summary("Removes specified roles to a user.")]
         public async Task RemoveRolesAsync(IGuildUser user, params IRole[] roles)
         {
             await user.RemoveRolesAsync(roles);
-            await Context.Message.AddReactionAsync(new Emoji("✅"));
+
+            var embed = new EmbedBuilder()
+                .WithDescription($"Removed {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} from {user.Mention}.")
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:156]: `roles.Humanize()`

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 3]
```diff
@@ -141,7 +179,11 @@ public async Task RemoveRolesAsync(params SocketRole[] roles)
                 }
             }
 
-            await message.AddReactionAsync(new Emoji("✅"));
+            var embed = new EmbedBuilder()
+                .WithDescription($"Removed {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} from everyone.")
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:183]: `roles.Humanize()`

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 4]
```diff
@@ -257,12 +299,15 @@ private async Task ViewRolesInfoAsync(IEnumerable<SocketRole> roles)
...
+            [HelpSummary("Hoist the role in the member list")]
             public bool? IsHoisted { get; set; }
 
+            [HelpSummary("Make the role mentionable")]
             public bool? IsMentionable { get; set; }
 
+            [HelpSummary("Choose the color of the role")]
             public Color? Color { get; set; }
-
+            [HelpSummary("List of permissions")]
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:310]: Missing a blank line before the attribute here

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 5]
```diff
@@ -40,8 +41,16 @@ public Task AddRoleAsync(IGuildUser user, IRole role, TimeSpan? length = null)
         [Summary("Adds specified roles to a user.")]
         public async Task AddRolesAsync(IGuildUser user, params IRole[] roles)
         {
+
+
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:45]: Extra space, should be removed

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 6]
```diff
@@ -40,8 +41,16 @@ public Task AddRoleAsync(IGuildUser user, IRole role, TimeSpan? length = null)
...
             await user.AddRolesAsync(roles);
-            await Context.Message.AddReactionAsync(new Emoji("✅"));
+
+            var embed = new EmbedBuilder()
+                .WithDescription($"Added {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} to {user.Mention}.")
+                .WithColor(Color.Green);
+
+            await ReplyAsync(embed: embed.Build());
+
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:53]: Extra space, should be removed

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 7]
```diff
@@ -65,7 +74,12 @@ public async Task AddRolesAsync(params SocketRole[] roles)
                 }
             }
 
-            await message.AddReactionAsync(new Emoji("✅"));
+            var embed = new EmbedBuilder()
+                .WithDescription($"Added {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} to everyone.")
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:78]: `roles.Humanize()`

File: Zhongli.Bot/Modules/RoleModule.cs [hunk 8]
```diff
@@ -40,8 +41,16 @@ public Task AddRoleAsync(IGuildUser user, IRole role, TimeSpan? length = null)
         [Summary("Adds specified roles to a user.")]
         public async Task AddRolesAsync(IGuildUser user, params IRole[] roles)
         {
+
+
             await user.AddRolesAsync(roles);
-            await Context.Message.AddReactionAsync(new Emoji("✅"));
+
+            var embed = new EmbedBuilder()
+                .WithDescription($"Added {Format.Bold(String.Join(separator: ", ", values: roles.Select(r => r.ToString())))} to {user.Mention}.")
```
Comment (sabihoshi) [inline Zhongli.Bot/Modules/RoleModule.cs:49]: Use `roles.Humanize()` here, it does the same thing essentially

**Review by sabihoshi — CHANGES_REQUESTED** [review]

**Review by sabihoshi — APPROVED** [review]

**Pull request merged**